FlowCrypt/Common/Services/Encrypted Storage/KeychainKeyGenerator.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  KeychainKeyGenerator.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 25.11.2019.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+protocol KeychainKeyGeneratorType {
+    func generateKeyData() -> Data
+}
+
+struct KeychainKeyGenerator: KeychainKeyGeneratorType {
```

Comment:
Instead of this you can use `CoreHost.getSecureRandomByteNumberArray(64)`, and convert that to `Data` if needed.

Reply:
Fixed